src/Common/tests/System/Collections/TestingTypes.cs

```diff
@@ -194,6 +197,9 @@ public int GetHashCode(IEqualityComparer comparer)
         }
```

Comment:
I didn't remove them in the tests. This is an existing type in the tests that now needs to be serializable. Making it a partial class to add the attribute in another file seems like overkill.